src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,52 @@
+-module(couch_replicator_tasks).
+
+-export([all/0]).
+
+-include_lib("couch/include/couch_db.hrl").
+-include("couch_replicator.hrl").
+-include("couch_replicator_api_wrap.hrl").
+
+
+all() ->
+    Jobs = ets:tab2list(couch_replicator_scheduler),
+    lists:map(fun(Job) ->
```

Comment:
Minor nit. If ets gets larger, wonder if ets:foldl will be more efficient. It would avoid one extra full copy of the ets table as it would process records one at  a time. I can see how tab2list is cleaner in code though....